resources/dicts/thoughts/alive/warrior.json

```diff
@@ -410,7 +409,8 @@
         "thoughts": [
             "Is scolding the apprentices over something slight",
             "Is pushing around the apprentices",
-            "Berates an apprentice for not doing their chores"
+            "Berates an apprentice for not doing their chores",
+            "Tells r_c to toughen up or else"
```

Comment:
This is a bit threatening, almost edging on truly evil. It might be better to soften it to "Tell r_c to toughen up"